src/spikeinterface/core/generate.py

```diff
@@ -984,8 +984,8 @@ class NoiseGeneratorRecording(BaseRecording):
         The sampling frequency of the recorder.
     durations : List[float]
         The durations of each segment in seconds. Note that the length of this list is the number of segments.
-    noise_level: float, default: 1
-        Std of the white noise
+    noise_levels: float or array, default: 1
+        Std of the white noise (if an array, defined by per channels)
```

Comment:
Can you add a test for this new behavior?

Reply:
A test has been added in test_generate.py